MetaMask/metamask-extension
# Adding HelpText component

## Explanation

Adding `HelpText` component that will eventually be used with `TextField` and `Label`  to create `FormTextField`.

## More Information

* Fixes #16151
* [Figma Link](https://www.figma.com/file/HKpPKij9V3TpsyMV1TpV7C/DS-Components?node-id=7294%3A25180)

## Screenshots/Screencaps

### After


https://user-images.githubusercontent.com/8112138/198402952-2db1b8a2-62c6-4a40-b962-d417014d1d05.mov



## Manual Testing Steps

- Go to the latest build of Storybook in this PR
- Search `HelpText` in the search bar
- Check story controls
- Check stories
- Check docs

## Pre-Merge Checklist


### + If there are functional changes:


### Acceptance Criteria
- [x] Standardize all similar prop names for images `imgSrc`, `imgAlt`(html element + attribute) (needs audit)
